doc-content/jbpm-docs/src/main/asciidoc/BPMN2/resize-elements-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='resize-elements']
+
+= Resizing elements
+You can resize individual elements and finished business processes.
+
+.Procedure
+. Select the element and click on the red dot on the corner of the element.
+. Drag the red dot to resize the element.
++
+.Resize an element
+image::processes/red-dot-resize.png[Resizing an element]
+. To resize the entire diagram, click the plus or minus sign on the lower right side of the canvasl
```

Comment:
well in fact is not correct saying that the diagram is being resized. In fact the diagram (process) cannot be resized, there is no "size" for the process, rather than for an element, which can have some size and also be resized (user tasks, etc)
When cliking on the below component what you achieve is different zoom levels, but the diagram is never being reiszed in fact, I mean, once you click on the "+" or "-", the whole zoom changes, but none of the elements ins the process are being resized.